### PR TITLE
Add skills PPLTool execution cluster setting

### PR DESCRIPTION
**Pull request by zane-neo**
### Description
Add skills PPLTool execution cluster setting

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Version
_List the OpenSearch version to which this PR applies, e.g. 2.14, 2.12--2.14, or all._

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
@zane-neo - Is this new for 2.16 or something that was missed before?

**Comment by zane-neo** [comment]
> @zane-neo - Is this new for 2.16 or something that was missed before?

This is for 2.16.

File: _ml-commons-plugin/cluster-settings.md [hunk 1]
```diff
@@ -492,3 +492,14 @@ plugins.ml_commons.rag_pipeline_feature_enabled: true
 
 - Default value: `true`
 - Valid values: `false`, `true`
+
```
Comment (kolchfa-aws) [inline _ml-commons-plugin/cluster-settings.md:495]: Please add a heading level 2 before this section that says what the setting does and a description of the setting.

**Review by kolchfa-aws — CHANGES_REQUESTED** [review]
Please add a name/description for this setting before line 495.

File: _ml-commons-plugin/cluster-settings.md (hunk 1 above)
Comment (zane-neo) [reply _ml-commons-plugin/cluster-settings.md:495]: Thanks, add description.

**Comment by zane-neo** [comment]
Please hold on the merge, confirming if this PR is necessary.

**Comment by zane-neo** [comment]
This is not necessary since the cluster setting code has been reverted: https://github.com/opensearch-project/skills/pull/381, closing it now.

**Pull request closed without merging**